package.json

```diff
@@ -3,20 +3,24 @@
   "version": "1.1.0",
   "description": "Flux Standard Action utlities for Redux",
   "main": "lib/index.js",
+  "module": "es/index.js",
```

Comment:
`module` serves about the same purpose as `jsnext:main` and is used by [webpack2](https://webpack.js.org/guides/author-libraries/#final-steps) and [Rollup](https://github.com/rollup/rollup-plugin-node-resolve#readme). I went by [this Rollup wiki page](https://github.com/rollup/rollup/wiki/jsnext:main) which says

> If in doubt, use both!